content/zh-cn/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -92,13 +92,13 @@ Cluster administrator actions include:
 
 - [Draining a node](/docs/tasks/administer-cluster/safely-drain-node/) for repair or upgrade.
 - Draining a node from a cluster to scale the cluster down (learn about
-[Cluster Autoscaling](https://github.com/kubernetes/autoscaler/#readme)).
+[Node Autoscaling](/docs/concepts/cluster-administration/node-autoscaling/)).
 - Removing a pod from a node to permit something else to fit on that node.
 -->
 集群管理员操作包括：
 
 - [排空（drain）节点](/zh-cn/docs/tasks/administer-cluster/safely-drain-node/)进行修复或升级。
-- 从集群中排空节点以缩小集群（了解[集群自动扩缩](https://github.com/kubernetes/autoscaler/#readme)）。
+- 从集群中排空节点以缩小集群（了解[节点自动扩缩](/zh-cn/docs/concepts/cluster-administration/node-autoscaling/)）。
```

Comment:
https://deploy-preview-49980--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/cluster-administration/node-autoscaling/ page is 404.